lib/ansible/modules/database/proxysql/proxysql_query_rules.py

```diff
@@ -513,16 +541,20 @@ def main():
             match_digest=dict(type='str'),
             match_pattern=dict(type='str'),
             negate_match_pattern=dict(type='bool'),
+            re_modifiers=dict(type='str'),
```

Comment:
rather than use `type='str'`, for a "comma separate list" i would turn this option to a type=`list` in the arg spec and make it comma separated by `','.join(re_modifiers)` it.

This allows to use it (due to ansible magic) like `re_modifiers=one,tow,three` as well as `re_modifiers: [ one, two, three ]`

Reply:
good point!